sources/tools/Stride.TextureConverter/Frontend/TextureTool.cs

```diff
@@ -40,8 +40,9 @@ static TextureTool()
             NativeLibraryHelper.PreloadLibrary("DxtWrapper", type);
             NativeLibraryHelper.PreloadLibrary("PVRTexLib", type);
             NativeLibraryHelper.PreloadLibrary("PvrttWrapper", type);
-            NativeLibraryHelper.PreloadLibrary("FreeImage", type);
-            NativeLibraryHelper.PreloadLibrary("FreeImageNET", type);
+            NativeLibraryHelper.PreloadLibrary("freeimage", type);
+            //TODO: needs to explain why FreeImageNET is loaded as a dll instead of directly referencing the C# project (this does not affect the compilation process on Linux).
+            NativeLibraryHelper.PreloadLibrary("FreeImageNET", type, !OperatingSystem.IsWindows()); 
```

Comment:
If condition here instead of making a useless call that returns immediately.

Reply:
I think your point is valid, fixed!